index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css" >
+</head>
+<body>
+  <header class="top-bar">
+    <img class="facebook-logo" src="imgs/facebook-logo.png" alt="Logo Facebook">
+    <form class="facebook-login" action="" method="GET">
+      <div class="form-group">
+        <label for="user-email-phone" id="user-email-phone-label">Email ou telefone</label>
+        <input type="text" id="user-email-phone" name="user-email-phone">
+      </div>
+      <div class="form-group">
+        <label for="user-password" id="user-password-label">Senha</label>
+        <input type="password" id="user-password" name="user-password">
+      </div>
+      <div class="form-control">
+        <button id="button-login" type="submit" >Entrar</button>
```

Comment:
O que voce acha de remover esse espaço?
```suggestion
        <button id="button-login" type="submit">Entrar</button>
```